pandas/tests/series/test_repr.py

```diff
@@ -198,6 +198,13 @@ def test_latex_repr(self):
 
         assert s._repr_latex_() is None
 
+    def test_categorical_index_repr_in_frame_with_nan(self):
+        i = Index([1, np.nan])
```

Comment:
Reference issue above this line as a comment.

Reply:
I meant above the line `i = Index(...)` (hence why I put the comment where I did).  I corrected it myself.